armtemplates/template.bicep

```diff
@@ -0,0 +1,280 @@
+param virtualMachines_iagora_name string = 'iagora'
+param publicIPAddresses_iagora_ip_name string = 'iagora-ip'
+param virtualNetworks_iagora_vnet_name string = 'iagora-vnet'
+param networkInterfaces_iagora731_z1_name string = 'iagora731_z1'
+param networkSecurityGroups_iagora_nsg_name string = 'iagora-nsg'
+param sshPublicKey string 
+param vmSize string = 'Standard_B2s' 
+
+//NSG ressourc
+resource networkSecurityGroups_iagora_nsg_name_resource 'Microsoft.Network/networkSecurityGroups@2023-11-01' = {
+  name: networkSecurityGroups_iagora_nsg_name
+  //name:'iagora-nsg'
+  location: 'francecentral'
+  properties: {
+    securityRules: [
+      {
+        name: 'SSH'
+        
+        properties: {
+          protocol: 'TCP'
+          sourcePortRange: '*'
+          destinationPortRange: '22'
+          sourceAddressPrefix: '*'
+          destinationAddressPrefix: '*'
+          access: 'Allow'
+          priority: 300
+          direction: 'Inbound'
+          sourcePortRanges: []
+          destinationPortRanges: []
+          sourceAddressPrefixes: []
+          destinationAddressPrefixes: []
+        }
+      }
+      {
+        name: 'HTTP'
+        //id: networkSecurityGroups_iagora_nsg_name_HTTP.id
+        //type: 'Microsoft.Network/networkSecurityGroups/securityRules'
```

Comment:
```suggestion
```
If the purpose of a comment is to remove code, why should we keep it?

Reply:
well done !!

Reply:
If you agree, could you then please remove these comments?